HTMLToQPDF/Utils/FontFamilyUtils.cs

```diff
@@ -10,6 +10,7 @@ internal static class FontFamilyUtils
             "Arial",
             "Brush Script MT",
             "Courier New",
+            "Garamond",
```

Comment:
what is this add for?

Reply:
This was a font I missed when adding the fonts in originally